common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py

```diff
@@ -191,15 +204,18 @@ def test_mongo_modulestore_type(self):
     def test_get_courses(self):
         '''Make sure the course objects loaded properly'''
         courses = self.draft_store.get_courses()
-        assert_equals(len(courses), 6)
+        assert_equals(len(courses), 7)  # pylint: disable=no-value-for-parameter
```

Comment:
Why are there seven courses above - and only six explicitly checked against below?